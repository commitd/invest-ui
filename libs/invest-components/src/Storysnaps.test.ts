// import initStoryshots from '@storybook/addon-storyshots'

type GlobalWithAnimationFrame = NodeJS.Global & {
    requestAnimationFrame: (callback: () => {}) => void
}

const polyfilledGlobal = global as GlobalWithAnimationFrame
if (!polyfilledGlobal.requestAnimationFrame) {
    polyfilledGlobal.requestAnimationFrame = (callback: () => {}) => setTimeout(callback, 0)
}

// TODO
// initStoryshots()
test('Storybook disabled due to STORYBOOK_ENV error', () => {
    console.log('Storybooks disabled')
})