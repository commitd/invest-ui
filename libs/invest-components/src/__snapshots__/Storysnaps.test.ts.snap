// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots Layout TODO 1`] = `
<p>
  TODO
</p>
`;

exports[`Storyshots Login TODO 1`] = `
<p>
  TODO
</p>
`;

exports[`Storyshots Navbar TODO 1`] = `
<p>
  TODO
</p>
`;

exports[`Storyshots Welcome to Invest Components 1`] = `
<div>
  <p>
    Vessel Components contains reusable react components which are not tried to any particular plugin or the application framework.
  </p>
</div>
`;
